gradle-sls-packaging/src/main/resources/init.sh

```diff
@@ -25,7 +25,7 @@ is_process_service() {
   local PID=$1
   local SERVICE_NAME=$2
   # trailing '=' prevents a header line
-  ps -o command= $PID | grep -q "$SERVICE"
+  ps ewww -o command= $PID | grep -q "$SERVICE_NAME"
```

Comment:
I've tested this on Mac OS X Sierra, CentOS 6.8, and IBM AIX 7.1 and the `ewww` flag prevents truncation of results as expected.